specification/chaos/resource-manager/Microsoft.Chaos/preview/2024-03-22-preview/types/targets.json

```diff
@@ -0,0 +1,59 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Target Types",
+    "version": "2024-03-22-preview"
+  },
+  "paths": {},
+  "definitions": {
+    "target": {
+      "type": "object",
+      "description": "Model that represents a Target resource.",
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/Resource"
```

Comment:
>Resource [](http://example.com/codeflow?start=97&length=8)

Shouldn't this be either ProxyResource or TrackedResource?

Reply:
[ARMBlockingComment] Given that it appears to be an extension resource, I think it has to be ProxyResource.

Reply:
updated to proxy resource